### PR TITLE
Update index.md : #### host or none - network

### PR DESCRIPTION
**Pull request by zeniri**
Choosing specific network for a build instead of the [network_mode]. network_mode doesn't work when providing a network for a particular build rather it skips the block and move to next service thus using network.

### Proposed changes
If in the compose file one is to provide a network for a particular build one can't do that. The "network_mode" will skip the block going over to the next block in the compose file itself and "networks" isn't helpful either. Thus using "network" tag that allows to provide network for every single build and solves the problem.

### CODE REVIEW

**Comment by L-Hudson** [comment]
@quadespresso PTAL thank you!

**Comment by zeniri** [comment]
@bermudezmt  PTAL thanks!

**Comment by bermudezmt** [comment]
Hi, @chris-crone - can you please verify the addition to HOST OR NONE here? Thanks ahead!

File: compose/compose-file/index.md [hunk 1]
```diff
@@ -2319,7 +2322,22 @@ networks:
...
+      ...
+```
+
+```yaml
+services:
+  web:
+    ...
+
     # ...
+
```
[the remark this answers is not in the document]
Comment (ulyssessouza) [reply compose/compose-file/index.md]: BTW, the comment "# ..." in the first yaml file would be better as just "..." to keep the uniformity with the rest.

**Comment by ulyssessouza** [comment]
Now you just need to squash your commits in just one. 
For the rest, it LGTM

**Comment by zeniri** [comment]
Hi @ulyssessouza, I did the commits directly from GitHub web UI which wasn't like what I did with githb console locally, and via web I don't have access to the option ' merge and squash ' , could you please help me check that ? Let me know in case of any queries. I had been really busy lately sorry to keep you waiting for my pull request, will do my best to contribute it.

image

**Comment by bermudezmt** [comment]
I've squashed and merged on your behalf, @zeniri . Thank you for your contribution!

**Comment by zeniri** [comment]
Thank you so much @bermudezmt for the help. I am glad I could contribute something like this.

**Pull request merged**